src/applications/disability-benefits/686c-674-v2/config/chapters/report-add-a-spouse/current-marriage-information/helpers.jsx

```diff
@@ -1,44 +1,145 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 
 export const marriageTypeInformation = (
   <div>
     <p>
-      If you select a marriage type other than{' '}
-      <span className="vads-u-color--gray-dark vads-u-font-weight--bold">
-        Ceremonial,
-      </span>{' '}
-      or if you don’t live in the U.S. or a territory of the U.S., we require
-      additional evidence to establish a spouse.
+      Based on your answer, you’ll need to submit additional documents to add
+      your spouse as a dependent.
     </p>
-    <va-additional-info
-      trigger="Additional evidence needed to add spouse"
-      uswds="false"
-    >
-      <p>
-        Examples of additional information includes a copy of the public record,
-        or a copy of the church record that shows:
-      </p>
-      <ul>
-        <li>The identities of the people involved</li>
-        <li>The date and place of the marriage</li>
-        <li>The number of prior marriages for both parties</li>
-      </ul>
-      <p>
-        If an official record is not available, you should submit at least one
-        of the following documents:
-      </p>
-      <ul>
-        <li>
-          The official report from your branch of service showing a marriage
-          happened while you were in service
-        </li>
-        <li>A certified copy of the original marriage certificate</li>
-        <li>
-          An affidavit of the clergyman or magistrate who conducted the marriage
-          ceremony.
-        </li>
-        <li>Other evidence that may be considered</li>
-      </ul>
-    </va-additional-info>
+    <p>We’ll ask you to submit these documents at the end of this form.</p>
   </div>
 );
+
+const baseMessage = (
+  <>
+    <p>
+      Based on your answer, you’ll need to submit additional documents to add
+      your spouse as a dependent.
+    </p>
+    <p>We’ll ask you to submit these documents at the end of this form.</p>
+  </>
+);
+
+const typeProxy = (
+  <>
+    <p>
+      You’ll need to submit copies of all documents and certificates issued in
+      connection with your proxy marriage.
+    </p>
+  </>
+);
+
+const typeCommonLaw = (
+  <>
+    <p>You’ll need to submit copies of all these documents:</p>
+    <ul>
+      <li>
+        Copies of the birth certificates for you and your spouse’s children,{' '}
+        <strong>and</strong>
+      </li>
+      <li>
+        2 Statements of Marital Relationship (VA Form 21-4170) — 1 that you
+        complete and 1 that your spouse completes, <strong>and</strong>
+        <br />
+        <a
+          href="/find-forms/about-form-21p-4170/"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          Get VA Form 21-4170 to download (opens in new tab)
```

Comment:
@DLarson-Oddball Not sure if i accidentally removed your comment here, but va-link was replaced by anchor tags because they do not support targets